bluemira/builders/tf_coils.py

```diff
@@ -598,7 +450,9 @@ class RippleConstrainedLengthGOP(GeometryOptimisationProblem):
     def __init__(
         self,
         parameterisation: GeometryParameterisation,
-        optimiser: Optimiser,
+        algorithm: str,
+        opt_conditions: Dict[str, float],
+        opt_parameters: Dict[str, float],
```

Comment:
Wee bit clunky methinks, especially as opt_parameters is usually {} which we can't have as a default argument, and would need to separate out to be below other non default arguments.

Reply:
Is this still an issue? I'm not sure that I see the problem with these having overridable defaults